### PR TITLE
Adjustments to Admin Panel + Bugfix

### PR DESCRIPTION
**Pull request by preimpression**
Moved several category links out of the sidebar and into their target indexes (aka item categories is now a link in items) to make the sidebar a little less Tall, as well as reorganized some of the Data sidebar links by purpose (shops and currencies together, prompts and loot tables together, etc)

### CODE REVIEW

**Review by itinerare — APPROVED** [review]

File: config/lorekeeper/admin_sidebar.php [hunk 1]
```diff
@@ -75,9 +75,42 @@
...
+                'name' => 'MYO Approvals',
+                'url' => 'admin/myo-approvals/pending'
+            ],
+        ]
+    ],
     'Data' => [
         'power' => 'edit_data',
         'links' => [
+            [
+                'name' => 'Char. Categories',
```
Comment (Draginraptor) [inline config/lorekeeper/admin_sidebar.php:111]: Not really a fan of this being the sole abbreviation of character in the sidebar

**Review by Draginraptor — COMMENTED** [review]
Just thinking about 'Char. Categories' being the only instance of 'Character' that's abbreviated xD

File: config/lorekeeper/admin_sidebar.php (hunk 1 above)
Comment (preimpression) [reply config/lorekeeper/admin_sidebar.php:111]: It was the longest thing on the sidebar after removing Categories, it just makes the sidebar so wide haha. I can remove it if the consensus is back to normal though

Comment (Draginraptor) [reply config/lorekeeper/admin_sidebar.php:111]: Personally in favour of maintaining consistency with "Character", but maybe @itinerare has an opinion? :p

Comment (itinerare) [reply config/lorekeeper/admin_sidebar.php:111]: I don't personally see an issue w it but I also understand the feeling toward consistency.... I don't have a strong opinion either way though |D

Comment (preimpression) [reply config/lorekeeper/admin_sidebar.php]: Made this change @Draginraptor !

**Review by itinerare — APPROVED** [review]

**Review by Draginraptor — APPROVED** [review]
Apologies for the late reply! But yes, my desire for consistency has be satisfied :p

**Pull request merged**